### PR TITLE
Fix #762: Add a column for OTP attempt count

### LINKED ISSUE #762: Add a column for OTP attempt count
Currenly only failed attempt count is stored into the database. For reporting purposes it is useful to know all attempt count, so a separate column with verification attempt counter would be useful.

### PR DESCRIPTION
**Pull request by jandusil**
* Update table es_onboarding_otp -> new col 'total_attempts'
* Update 'total_attempts' count during various processes

### CODE REVIEW

**Comment by romanstrobl** [comment]
The commits do not seem to be signed. Please make sure to sign any future commits, and let's merge this PR using merge commit after signing is configured, so that the final commit is signed. Thanks.

File: docs/db/changelog/changesets/enrollment-server-onboarding/1.6.x/20230530-add-column-total-attempts.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="enrollment-server/1.6.x/20230530-add-column-total-attempts.xml" author="Jan Dusil">
```
Comment (banterCZ) [inline docs/db/changelog/changesets/enrollment-server-onboarding/1.6.x/20230530-add-column-total-attempts.xml:6]: Target version is 1.5.0

File: docs/db/changelog/changesets/enrollment-server-onboarding/1.6.x/20230530-add-column-total-attempts.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="es_onboarding_otp" columnName="total_attempts"/>
+            </not>
+        </preConditions>
+        <comment>Add total_attempts column</comment>
+        <addColumn tableName="es_onboarding_otp">
+            <column name="total_attempts" type="integer" />
```
Comment (banterCZ) [inline docs/db/changelog/changesets/enrollment-server-onboarding/1.6.x/20230530-add-column-total-attempts.xml:14]: What will be the default value? Are we ok with `null`? The application is already deployed in production.

File: docs/sql/oracle/onboarding/create-schema.sql [hunk 3]
```diff
@@ -51,6 +51,7 @@ CREATE TABLE ES_ONBOARDING_OTP (
     ERROR_DETAIL VARCHAR2(256 CHAR),
     ERROR_ORIGIN VARCHAR2(256 CHAR),
     FAILED_ATTEMPTS INTEGER,
+    TOTAL_ATTEMPTS INTEGER,
```
Comment (banterCZ) [inline docs/sql/oracle/onboarding/create-schema.sql:54]: Should be also described at `Database-Structure.md`.

**Review by banterCZ — COMMENTED** [review]
Very good attempt for the first PR.

**Pull request closed without merging**